### PR TITLE
Adds !trades command

### PR DESCRIPTION
**Pull request by Waflson**
Trades done in the last 24 hours & total trades recorded.

### CODE REVIEW

**Comment by Nicklason** [comment]
https://github.com/Nicklason/tf2-automatic/issues/177

File: app/handler/commands.js [hunk 1]
```diff
@@ -657,6 +658,25 @@ exports.handleMessage = function (steamID, message) {
...
+        const offerData = manager.pollData.offerData
+
+        let tradeCount = 0;
+        let tradeTotal = 0;
+        for (const offerID in offerData) {
+            if (!Object.prototype.hasOwnProperty.call(offerData, offerID)) {
+                continue;
+            }
+
+            if (offerData[offerID].finishTimestamp >= (dateNow - (86400000)) && offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) { // Sucessful trades accepted in the last 24 hours by bot
```
Comment (Nicklason) [inline app/handler/commands.js]: Should probably move the comment to the next line

File: app/handler/commands.js [hunk 2]
```diff
@@ -657,6 +658,27 @@ exports.handleMessage = function (steamID, message) {
...
+            }
+
+            if (offerData[offerID].finishTimestamp >= (dateNow - (86400000)) && offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) { 
+                // Sucessful trades accepted in the last 24 hours by bot
+                tradeCount++;
+                tradeTotal++;
+            } else if (offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) { 
+                // Successful trades not accepted in the last 24 hours by bot
+                tradeTotal++;
+            }
```
Comment (Nicklason) [inline app/handler/commands.js]: Duplicate code. First check if the offer is accepted and handled by us, then increment tradeTotal. If it is within 24 hours then add it to tradeCount

**Comment by Hhanuska** [comment]
You removed spaces after function names, which breaks ["space-before-function-paren"](https://github.com/Nicklason/tf2-automatic/blob/master/.eslintrc#L22) eslint rule.
https://eslint.org/docs/rules/space-before-function-paren

File: app/handler/commands.js [hunk 3]
```diff
@@ -657,6 +658,28 @@ exports.handleMessage = function (steamID, message) {
...
+        for (const offerID in offerData) {
+            if (!Object.prototype.hasOwnProperty.call(offerData, offerID)) {
+                continue;
+            }
+
+            if (offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) {
+                // Sucessful trades handled by the bot
+                tradeTotal++;
+
+                if (offerData[offerID].finishTimestamp >= (dateNow - 86400000)) {
```
Comment (danocmx) [inline app/handler/commands.js]: Make day start and end so we get accurate data to current day.

Comment (Nicklason) [reply app/handler/commands.js]: It is better to just do it over the last 24 hours, it is easier for people to read and it does not change depending on the time off the day.

**Pull request merged**